qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -281,7 +273,7 @@ def status(self) -> BackendStatus:
 
         try:
             return BackendStatus.from_dict(api_status)
```

Comment:
Yes `ModelValidationError` is a class in terra that's being removed by Qiskit/qiskit-terra#4030. It's preemptively removed here because the odds of someone using terra master with `ModelValidationError` removed and ibmq-provider that still uses it, is much greater than a `ModelValidationError` actually being raised.